openwisp_monitoring/monitoring/static/monitoring/js/chart.js

```diff
@@ -355,8 +362,11 @@
                 }
                 percircles.push(percircleOptions);
 
-                if (unit == 'adaptive_bytes') {
-                    adaptiveFilterSummary(i, percircles, value);
+                if (unit.includes('adaptive_prefix')) {
+                    if(unit.includes('+')){
```

Comment:
respect the spacing (I wish the linter warned about this!):

```
if (unit.includes('+')) {
`

Reply:
Linter didn't warn 🤔 I'll check all my code manually :+1: 

![Screenshot from 2022-08-04 21-54-41](https://user-images.githubusercontent.com/56113566/182901953-77fbb3ee-110b-4660-ba2b-c2d5fdb60f2b.png)